### ISSUE TITLE
Switching the organization password type should re-create the passwords for the organization

### ISSUE THREAD

**Issue by idc77** [casdoor/casdoor#3278]
In the following scenario, I have changed the default built-in organization's password type from plain to bycrypt.
The 1st user, Admin, is part of that Org.
After changing the password type, the plaintext password remained stored in the database.

Danger of locking oneself out.

In the case of the password being plain, it should be hased with the new password type.
There's a catch 22 situation when you switch from non-plain to other non-plain password types.
When switching password types, there should be a warning message, that the admin user should modify their password.
Of course it's logical to do so, but I don't believe this behavior is documented.

Of course how often does it happen that the password type changes? Probably once for the built-in org.
Speaking of, the default password type for the built-in org should be set to argon2d.
Storing passwords in plaintext is a liability in at least 1 country on the planet and can cost a lot of money in case of a breach.

So, TL;DR
if previous password type == plain, automatically generate new password hashes for all users of the org
if new password type == plain, since we don't know the unhashed password, do nothing.
Make argon2d the new default password type for the built-in organization

**Comment by hsluoyz** [comment]
It's better if you can contribute to this issue. It will be low priority on our side.

Closed for now. If a promissing PR is made, it may be opened again
